addons/l10n_ec/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="res_config_settings_view_form" model="ir.ui.view">
+            <field name="name">res.config.settings.view.form.inherit.account</field>
+            <field name="model">res.config.settings</field>
+            <field name="priority" eval="40"/>
+            <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[@name='fiscal_localization_setting_container']" position="after">
+                    <h2 attrs="{'invisible': [('country_code', '!=', 'EC')]}">Electronic Documents</h2>
```

Comment:
Is there a point in storing the certificates and not sending the electronic documents to the government?  Seems like this should reside in the other module.